src/lambdas/downloadWarning/handler.ts

```diff
@@ -22,11 +28,13 @@ export const handler = async (
       return notFoundResponse()
     }
 
+    logger.info('Data persistence test log')
```

Comment:
I presume you don't want to leave this in?

Reply:
removed